### PR TITLE
In LLMChain include format_instructions when formatting the prompt if the prompt accepts it

### PR DESCRIPTION
**Pull request by nfcampos**

### CODE REVIEW

File: langchain/src/chains/tests/llm_chain.test.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+  expect(res).toEqual({ [chain.outputKey]: "a completion" });
+  expect(model._call).toHaveBeenCalledWith(
+    "Print my favorite color",
+    undefined
+  );
+});
+
+test("Test with output_parser without format_instructions", async () => {
+  const model = new FakeLLM();
```
Comment (agola11) [inline langchain/src/chains/tests/llm_chain.test.ts:65]: Should we consider throwing an error if prompt has an output parser but no format_instructions variable?

**Review by agola11 — APPROVED** [review]
This is to get around needing to use `partialValues` right? Makes sense to me

File: langchain/src/chains/tests/llm_chain.test.ts (hunk 1 above)
Comment (nfcampos) [reply langchain/src/chains/tests/llm_chain.test.ts:65]: I guess we could, but would be less backwards compatible

**Comment by nfcampos** [comment]
note to me: do same in python

**Pull request closed without merging**